### DISCUSSION TITLE
Meta modifier for keyboard events doesn't seem to be working in firefox

### DISCUSSION THREAD

**Discussion by emil-nasso** [alpinejs/alpine#2585] [7. Bugs (archived)]
As the title is stating, I can't get the meta modifier to work for keyboard events in firefox. I'm running this in linux.

An example: https://jsfiddle.net/64gofxap/

If I click the resulting website and press meta (windows-key, meta, super, whatever you wanna call it) and the k-key at the same time, the counter is increased, when I use chrome. When I use firefox, nothing happens.

Has anyone else experienced this problem?

**Comment by mynona** [comment]
Doesn't work in Safari 18.3 neither.

**Comment by ekwoka** [comment]
Have you verified the event is firing at all?

**Comment by mynona** [comment]
yes, you can also check the firework fiddle from emil-nasso above

**Comment by ekwoka** [comment]
I tested it, and when the meta key is held, there is no keyup being triggered at all.

If you use `keydown` it will work.

**Comment by mynona** [comment]
Thank you. This workaround works.
